app/BUILD.bazel

```diff
@@ -138,6 +140,7 @@ DATABINDING_LAYOUTS = ["src/main/res/layout*/**"]
 VIEW_MODELS_WITH_RESOURCE_IMPORTS = [
     "src/main/java/org/oppia/android/app/help/HelpItemViewModel.kt",
     "src/main/java/org/oppia/android/app/help/HelpListViewModel.kt",
+    "src/main/java/org/oppia/android/app/help/HelpViewModel.kt",
```

Comment:
I think this can go in the regular VIEW_MODELS list since this doesn't actually depend on R.

Reply:
Done.